Detectors/TRD/qc/src/Tracking.cxx

```diff
@@ -134,7 +134,9 @@ void Tracking::checkTrack(const TrackTRD& trkTrd, bool isTPCTRD)
     };
 
     // Corrected Tracklets
-    auto cor = mLocalGain.getValue(tracklet.getHCID() / 2, tracklet.getPadCol(), tracklet.getPadRow()) * (std::abs(tracklet.getSlopeBinSigned()) + 1.f);
+    auto tphi = std::tan(std::asin(tracklet.getSnp()));
```

Comment:
```suggestion
    auto tphi = tracklet.getSnp() / std::sqrt((1. - tracklet.getSnp()) * (1. + tracklet.getSnp()));
```
To avoid expensive std::asin operation. Also, I think you need to check the track instead of the tracklet